app/src/sharedTest/java/org/oppia/android/app/player/exploration/ExplorationActivityTest.kt

```diff
@@ -1839,7 +1867,8 @@ class ExplorationActivityTest {
 
   private fun addShadowMediaPlayerException(dataSource: Any, exception: Exception) {
     val classLoader = ExplorationActivityTest::class.java.classLoader!!
-    val shadowMediaPlayerClass = classLoader.loadClass("org.robolectric.shadows.ShadowMediaPlayer")
+    val shadowMediaPlayerClass =
+      classLoader.loadClass("org.robolectric.shadows.ShadowMediaPlayer")
```

Comment:
Why this change? Ditto above. I suspect these might be extra and unnecessary reformats done by Android Studio.